region/maridia/inner-pink.json

```diff
@@ -7100,6 +7189,78 @@
                     "A diagonal ShineSpark will most likely not have any horizontal movement when performed from the sand."
                   ]
+                },
+                {
+                  "name": "G-Mode Through Crumbles",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }},
+                    "Gravity",
+                    "Morph"
+                  ],
+                  "note": "Get to the left side of the room while avoiding the hole in the ground, jump up through the Morph tunnel and Crumble Blocks."
+                },
+                {
+                  "name": "G-Mode Morph Through Crumbles",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Gravity",
+                    {"or":[
+                      "SpringBall",
+                      {"and":[
+                        "h_canArtificialMorphBombHorizontally",
+                        "h_canArtificialMorphIBJ"
+                      ]}
+                    ]}
+                  ],
+                  "note": [
+                    "Get to the left side of the room while avoiding the hole in the ground with Spring Ball or a carefully placed horizontal bomb boost.",
+                    "Use Spring Ball or an IBJ to get up through the Morph tunnel and Crumble Blocks."
+                  ]
+                },
+                {
+                  "name": "G-Mode Suitless Through Crumbles",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }},
+                    "canSuitlessMaridia",
+                    "Morph",
+                    "HiJump",
+                    "h_canCrouchJumpDownGrab"
+                  ],
+                  "note": "Get to the left side of the room while avoiding the hole in the ground, jump up through the Morph tunnel and Crumble Blocks."
+                },
+                {
+                  "name": "G-Mode Morph Suitless Through Crumbles",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "canSuitlessMaridia",
+                    "HiJump",
```

Comment:
A Morphless strat isn't relevant unless it can reach the item nodes but there is a shot block in the way.  Jumping from sand and a springballjump should be easier than gmode but it is a different set of tech 🤷‍♂️ 

There is value in a hjbless+springball+morph strat to 7.

Reply:
Spinjumping out of the sand works.  And springballjumps are enough to get to the items.

GMode+HiJump+Morph would avoid a `canBreakFree` 🤔